homework/Vasili_Zasinets/ArrayServiceTask/src/test/java/com/tutrit/java/quickstart/homework/arrayservice/ArrayServiceTest.java

```diff
@@ -0,0 +1,33 @@
+package com.tutrit.java.quickstart.homework.arrayservice;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ArrayServiceTest {
+    public int[] array;
+    ArrayService arrayService = new ArrayService();
+
+    @Before
+    public void setUp() {
+        this.array = arrayService.create(10);
```

Comment:
You create an array with test data in each test anyway, this could be deleted.

Reply:
ok